content/participate/report-issue.adoc

```diff
@@ -0,0 +1,215 @@
+
+Other types (_Patch_, _Task_) are not widely used in JIRA. Neither will
+be explained here. If you want to propose a patch, create an issue of
+one of the 3 types above and then create a pull request in GitHub.
+
+[[Howtoreportanissue-Beforereportingabug]]
+=== Before reporting a bug
+
+* First, *search for the error message or general problem description*.
+This can help you find solutions if it's actually e.g. a configuration
+problem, or point you to earlier reports in case you're not the first
+one experiencing this problem. You can
+http://issues.jenkins-ci.org/secure/IssueNavigator.jspa[use the search
+in the issue tracker], or use web search.
+https://stackoverflow.com/tags/jenkins[Stack Overflow] may also be a
+good place to search. Any results will often point you to earlier issue
+reports if it's really a bug. You can then _watch them and vote for
+them_ in JIRA. Voting helps us determine the impact of bugs and
+prioritize them. _If you have doubts that the problem you're
+experiencing is the exact same one as you found, prefer to file a new
+issue (and mention the one(s) you found in the description)._
+* *Make sure that the problem actually is with Jenkins*. If your build
+seems to fail for no reason, try to build your project outside Jenkins,
+but in the same general environment (e.g. same machine, same user
+account, ...). If Jenkins interacts with other systems, make sure that
+they're working. If the connections between Jenkins master and slaves
+fail, make sure the network is reliable.
+* *Check the changelogs* to see whether the issue has been fixed in
+newer versions of the affected component. For Jenkins itself, the
+changelog is http://jenkins-ci.org/changelog[here]. Select "Upcoming
+changes" to view changes that have been implemented but not yet
+released. If you believe the problem to be in a plugin, go to the
+plugin's
+https://wiki.jenkins-ci.org/display/JENKINS/Plugins#Plugins-Pluginsbytopic[wiki
+page] to see its change log. The plugin's wiki page also be reached
+directly by clicking the plugin name in _Manage Jenkins » Manage Plugins
+» Installed_.
+
+If you're still unable to resolve the problem and cannot find existing
+issue reports or related fixes in newer versions of Jenkins or the
+plugins, file a *bug* in JIRA.
+
+[[Howtoreportanissue-Beforerequestinganimprovementorfeature]]
+=== Before requesting an improvement or feature
+
+* Check whether newer versions of Jenkins or the plugin in question have
+been released. They may have the feature already!
+* *Search JIRA for existing requests* for this feature. You can then
+_watch them and vote for them_. Voting helps us determine the demand for
+requests and prioritize them, so it's in your interest to actually find
+these rather than reporting a duplicate.
+
+If you cannot find anything, file an *improvement* or *new feature*
+request in JIRA.
+
+[.aui-icon .aui-icon-small .aui-iconfont-warning .confluence-information-macro-icon]#
+#
+
```

Comment:
Is visible when rendered in the page in my development environment, but does not make sense.  Recommend removing it.

![image](https://user-images.githubusercontent.com/156685/80031781-6c52ec00-84a7-11ea-8568-53990a4feb8e.png)


```suggestion
```